storage/src/vespa/storage/persistence/filestorage/mergestatus.cpp

```diff
@@ -3,11 +3,70 @@
 #include "mergestatus.h"
 #include <ostream>
 #include <vespa/log/log.h>
+#include <vespa/vespalib/stllike/hash_map.h>
+#include <cassert>
 
 LOG_SETUP(".mergestatus");
 
 namespace storage {
 
+namespace {
+
+/*
+ * Class for remapping bit masks from a partial set of nodes to a full
+ * set of nodes.
+ */
+class MaskRemapper
```

Comment:
@vekterli and @toregge : Consider moving this out of the cpp file and then creating explicit unit tests for it.